# Publish all joint states -----------------------------------
joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

# little joint -------------------------------------
joint_lf_little_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_lf_little
    pid: {p: 50.0, i: 0.0, d: 2.0}

joint_rf_little_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_rf_little
    pid: {p: 50.0, i: 0.0, d: 2.0}

joint_bf_little_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_bf_little
    pid: {p: 50.0, i: 0.0, d: 2.0}

joint_br_little_position_controller:
    type: effort_controllers/JointPositionController
    joint: joint_br_little
    pid: {p: 50.0, i: 0.0, d: 2.0}

# foot wheel ----------------------------------------
joint_lf_foot_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_lf_foot

joint_rf_foot_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_rf_foot

joint_bf_foot_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_bf_foot

joint_br_foot_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_br_foot

# arm joint ----------------------------------------
# joint1_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint1
#     pid: {p: 1000.0, i: 0.0, d: 10.0}

# joint2_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint2
#     pid: {p: 5000.0, i: 0.0, d: 40.0}

# joint3_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint3
#     pid: {p: 5000.0, i: 0.0, d: 40.0}

# joint4_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint4
#     pid: {p: 1000.0, i: 1.0, d: 10.0}

# joint5_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint5
#     pid: {p: 500.0, i: 0.0, d: 8.0}

# joint6_position_controller:
#     type: effort_controllers/JointPositionController
#     joint: joint6
#     pid: {p: 500.0, i: 0.0, d: 8.0}

# /myrobot/gazebo_ros_control:
#     pid_gains:
#         joint_lf_foot:
#             p: 0.0
#             i: 0.0
#             d: 0.0
#         joint_rf_foot:
#             p: 0.0
#             i: 0.0
#             d: 0.0
#         joint_bf_foot:
#             p: 0.0
#             i: 0.0
#             d: 0.0
#         joint_br_foot:
#             p: 0.0
#             i: 0.0
#             d: 0.0